src/content/docs/en/reference/container-reference.mdx

```diff
@@ -40,21 +40,21 @@ Enables rendering components using [HTML streaming](/en/guides/server-side-rende
 
 #### `renderers` option
 
-**Type:** `AstroRenderer[]`
+**Type:** `SSRLoadedRendere[]`
 
-A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script
+A list of client renderers required by the component. Use this if your `.astro` component renders any [UI framework components](/en/guides/framework-components/) using an official Astro integration (e.g. React, Vue, etc.). For each framework rendered, you must provide an object stating the integration package `name`, as well as both its client and server rendering script.
+
+In a vite-full environment, renderers should be loaded by using the function `loadRenderers`, from the virtual module `astro:container`.
```

Comment:
I'm not sure "a vite-full environment" is clear enough to be helpful. I'm not sure what specific conditions this does or doesn't include. Can you be more descriptive here? (Even to me, then I can put it into words if you like!)